pkg/clients/compute.go

```diff
@@ -29,19 +29,28 @@ import (
 	uflavors "github.com/gophercloud/utils/openstack/compute/v2/flavors"
 
 	"sigs.k8s.io/cluster-api-provider-openstack/pkg/metrics"
+	openstackutil "sigs.k8s.io/cluster-api-provider-openstack/pkg/utils/openstack"
 )
 
 /*
-NovaMinimumMicroversion is the minimum Nova microversion supported by CAPO
-2.60 corresponds to OpenStack Queens
+Constants for specific microversion requirements.
+2.60 corresponds to OpenStack Queens and 2.53 to OpenStack Pike,
+2.38 is the maximum in OpenStack Newton.
```

Comment:
Would it be possible to expand the comment with the reason on why 2.38 is the `MinimumNovaMicroversion`?

Reply:
Good point! I added a note about it now. It is not set in stone at all, for more details see the [discussion here above](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1567#discussion_r1210568162)